tests/test_pigeon.py

```diff
@@ -142,6 +142,60 @@ def test_get_and_write_file_data_to_temp(
             assert info == expected_json
 
 
+class TestContributors:
+
+    @pytest.fixture
+    def guid(self):
+        return 'ft3ae'
+
+    @pytest.fixture
+    def file_name(self):
+        return 'contributors.json'
+
+    @pytest.fixture
+    def contributors_file(self):
+        with open(os.path.join(HERE, 'fixtures/ft3ae-contributors.json'), 'r') as fp:
+            return fp.read()
+
+    @pytest.fixture
+    def institutions_file(self):
+        with open(os.path.join(HERE, 'fixtures/ft3ae-institutions.json'), 'r') as fp:
+            return fp.read()
+
+    def test_get_and_write_file_data_to_temp(
+            self,
+            mock_osf_api,
+            guid,
+            contributors_file,
+            institutions_file,
+            file_name):
+        mock_osf_api.add(
+            responses.GET,
+            f'{settings.OSF_API_URL}v2/registrations/{guid}/contributors/',
+            status=200,
+            body=contributors_file
+        )
+        mock_osf_api.add(
+            responses.GET,
+            'http://localhost:8000/v2/users/s3rbx/institutions/',
```

Comment:
nit: being inconsistent with the formatting, I'd stay away from expressing this as a literal when it's totally formated on https://github.com/CenterForOpenScience/osf-pigeon/pull/19/files#diff-f945dd962147336547d5bd4bfb4c558eR174